__tests__/server.test.js

```diff
@@ -2,10 +2,12 @@
 
 require('@code-fellows/supergoose');
 
+const User = require('../model/signin');
 const { server } = require('../server');
 const agent = supergoose(server);
```

Comment:
The supergoose isn't defined here, are you trying to use supertest? Then it would be:
```
const supergoose = require('supertest');
const agent = supergoose(server);
```